judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/AbstractTest.java

```diff
@@ -0,0 +1,533 @@
+     *
+     *  Check if the I and J instance are equals.
+     *
+     *  Update relations String fields value with the IDao.
+     *
+     *  Check the relations String fields updated in the h representation with the HDao.
+     *
+     *  Delete relations with the IDao.
+     *
+     *  Check if the relations are deleted in the j representation with the JDao.
+     *
+     *  Add relations with the HDao.
+     *
+     *  Check the relations added with the HDao.
+     *
+     *  Remove relations with the HDao.
+     *
+     *  Check if the relations are removed with the HDao.
+     *
+     *  Unset relations with the HDao.
+     *
+     *  Check if the relations are unset with the HDao.
+     *
+     */
+    @Test
+    @TestCase("AbstractInheritedEntityRelationsAndCompositionsOnANotAbstractEntity")
+    @Requirement(reqs = {
+            "REQ-TYPE-001",
+            "REQ-TYPE-004",
+            "REQ-ENT-001",
+            "REQ-ENT-002",
+            "REQ-ENT-004",
+            "REQ-ENT-005",
+            "REQ-ENT-007",
+            "REQ-ENT-012",
+            "REQ-MDL-001",
+            "REQ-MDL-002",
+            "REQ-SYNT-001",
+            "REQ-SYNT-002",
+            "REQ-SYNT-003",
+            "REQ-SYNT-004"
+    })
+    void testAbstractInheritedEntityRelationsAndCompositionsOnANotAbstractEntity() {
+
+        //Entity H,I,J
+
+        J jOptionalRelation = jDao.create(J.builder().build());
+        I iOptionalRelation = JtoI(jOptionalRelation);
+        J jRequiredRelation = jDao.create(J.builder().build());
+        I iRequiredRelation = JtoI(jRequiredRelation);
+
+        List<J> jMultiRelation = List.of(
+                jDao.create(J.builder().build()),
+                jDao.create(J.builder().build()),
+                jDao.create(J.builder().build())
+        );
+        List<I> iMultiRelation = jMultiRelation.stream().map(l -> JtoI(l)).collect(Collectors.toList());
+
+        H h = hDao.create(H.builder()
+                        .withCompositionIOnHSingle(JtoI(jDao.create(J.builder().build())))
+                        .withCompositionIOnHSingleRequired(JtoI(jDao.create(J.builder().build())))
+                        .withCompositionIOnHMulti(List.of(
+                                jDao.create(J.builder().withNameI("V").build()),
+                                jDao.create(J.builder().withNameI("V").build()),
+                                jDao.create(J.builder().withNameI("V").build())
+                        ).stream().map(l -> JtoI(l)).collect(Collectors.toList()))
+                        .build(),
+                HAttachedRelationsForCreate.builder()
+                        .withRelationIOnHSingle(iOptionalRelation)
+                        .withRelationIOnHSingleRequired(iRequiredRelation)
+                        .withRelationIOnHMulti(iMultiRelation)
+                        .build()
+        );
+
+        //Check j and i instance ar equal because i is live with j only
```

Comment:
I suggested a fix for a similar comment to this. Please fix this, too.

Applies to subsequent occurrences, please check other cases.